pkg/server/server.go

```diff
@@ -783,25 +801,23 @@ func (s *Server) handleOpenShiftTokenDeletion(user *auth.User, w http.ResponseWr
 		tokenName = tokenToObjectName(tokenName)
 	}
 
-	path := "/apis/oauth.openshift.io/v1/oauthaccesstokens/" + tokenName
-	// Delete the OpenShift OAuthAccessToken.
-	url := proxy.SingleJoiningSlash(s.K8sProxyConfig.Endpoint.String(), path)
-	req, err := http.NewRequest("DELETE", url, nil)
+	clientConfig := rest.AnonymousClientConfig(s.InternalProxiedK8SClientConfig)
+	clientConfig.Host = s.KubeAPIServerURL
+	clientConfig.BearerToken = user.Token
+
+	oauthClient, err := oauthv1client.NewForConfig(clientConfig)
 	if err != nil {
-		serverutils.SendResponse(w, http.StatusInternalServerError, serverutils.ApiError{Err: fmt.Sprintf("Failed to create token DELETE request: %v", err)})
+		// FIXME: handle error
```

Comment:
Why not return an error response here?

Reply:
the PR was originally just a PoC, it's not yet finished